docs/jwst/ramp_fitting/main.rst

```diff
@@ -28,65 +28,95 @@ using the array-based functionality of numpy.  The size of the block depends
+arrays in this product is the result for a given integration.  In this output
+product, the GROUPDQ data for a given integration is compressed into 2-D, which
+is then merged with the input 2-D PIXELDQ to create the output DQ array for each
+integration. The 3-D VAR_POISSON and VAR_RNOISE from an integration are
+calculated by averaging over the fit segments in the corresponding 4-D arrays.
+By default, the name of this output product is based on the name of the input
+file and will have the suffix '_rateints'; the user can override this name by
+specifying a name using the parameter int_name.
 
 
 A third, optional output product is also available and is produced only when
 the step parameter 'save_opt' is True (the default is False).  This optional
 product contains 4-D arrays called SLOPE, SIGSLOPE, YINT, SIGYINT, WEIGHTS,
-VAR_POISSON, and VAR_RNOISE which contain the slopes, uncertainties in the slopes, 
-y-intercept, uncertainty in the y-intercept, fitting weights, the variance of the 
-slope due to poisson noise only, and the variance of the slope due to read noise 
-only for each segment of each pixel. (Calculaton of the two variance arrays
-requires retrieving readnoise and gain values from their respective reference
-files.)  The y-intercept refers to the result of the fit at an exposure time of
-zero.  This product also contains a 3-D array called PEDESTAL, which gives the
-signal at zero exposure time for each pixel, and the 4-D CRMAG array, which
-contains the magnitude of each group that was flagged as having a CR hit.  By
-default, the name of this output file is based on the name of the input file and
-will have the suffix '_fitopt'; the user can override this name by specifying a
-name using the parameter opt_name.  In this optional output product, the pedestal
-array is calculated for each integration by extrapolating the final slope (the
-weighted average of the slopes of all of ramp segments in the integration) for
-each pixel from its value at the first group to an exposure time of zero. Any
-pixel that is saturated on the first group is given a pedestal value of 0. Before
-compression, the cosmic ray magnitude array is equivalent to the input SCI array
-but with the only nonzero values being those whose pixel locations are flagged in
-the input GROUPDQ as cosmic ray hits. The array is compressed, removing all
-groups in which all the values are 0 for pixels having at least one group with a
-non-zero magnitude. The order of the cosmic rays within the ramp is preserved.
+VAR_POISSON, and VAR_RNOISE which contain the slopes, uncertainties in the
+slopes, y-intercept, uncertainty in the y-intercept, fitting weights, the
+variance of the slope due to poisson noise only, and the variance of the slope
+due to read noise only for each segment of each pixel. (Calculation of the two
+variance arrays requires retrieving readnoise and gain values from their
+respective reference files.)  The y-intercept refers to the result of the fit
```

Comment:
Does the user need to know that readnoise and gain were used to calculate the variances?  A reader might wonder whether he/she would need the readnoise and gain reference files in order to calculate the variances from the information in the VAR_POISSON and VAR_READNOISE values.